common.mk

```diff
@@ -0,0 +1,18 @@
+##############################################################################
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##############################################################################
+
+D = /
+P = :
```

Comment:
You need to add special case for windows:
```
ifneq (,$(findstring Win,$(OS)))
	P=;
endif
```
And remove this from other place, such as line 35 of makegen.mk and line 97 of settings.mk.